src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Classification
 {
     internal sealed class ClassificationTypeFormatDefinitions
     {
-        #region Preprocessor Text 
+        #region Preprocessor Text
```

Comment:
Is IDE0055 disabled? It should have caught the extra space if enabled

Reply:
IDE0055 is enabled. The formatter currently fails to remove certain end-of-line whitespaces.